python/lsst/ctrl/bps/wms/htcondor/htcondor_service.py

```diff
@@ -239,7 +236,7 @@ def write(self, out_prefix):
         self.dag.write(out_prefix, "jobs/{self.label}")
 
 
-def translate_job_cmds(generic_workflow_job):
+def _translate_job_cmds(generic_workflow_job):
```

Comment:
I'm not quite convinced that you really need the leading underscore to emphasize the fact that this function is "private". The list of public objects in the module (`__all__`) does not include it (and others) anyway. I mean nothing is technically wrong with having the underscore, so you may leave it as is. It just feels like an overkill.

Reply:
I'm fine with the underscore. Python lets people import things that aren't in the `__all__` and the leading underscore really does tell people they get everything they deserve if they import it anyhow.